### PR TITLE
minor stylistic changes for a better daily Turkish 

### PR DESCRIPTION
**Pull request by hwpplayer1**
I tried to make sentences to be understandable for native Turkish speaking people

### CODE REVIEW

**Review by FoggyMtnDrifter — APPROVED** [review]

File: i18n/tr.toml [hunk 1]
```diff
@@ -3,19 +3,19 @@ other = "tr"
 
 [links]
   [links.get-involved]
-  other = "Katılın"
+  ot	her = "Bize Katılın" # Bize katılın daha samimi duruyor
```
Comment (domingoruiz) [inline i18n/tr.toml]: Bug

**Review by domingoruiz — CHANGES_REQUESTED** [review]
There is a small bug, for the rest good.

File: i18n/tr.toml (hunk 1 above)
Comment (FoggyMtnDrifter) [reply i18n/tr.toml]: Good call out! This will need to be fixed before it can be merged.

Comment (FoggyMtnDrifter) [reply i18n/tr.toml]: @hwpplayer1 Go ahead and get this fixed and we can proceed

**Review by FoggyMtnDrifter — CHANGES_REQUESTED** [review]

**Review by FoggyMtnDrifter — APPROVED** [review]

File: i18n/tr.toml (hunk 1 above)
Comment (hwpplayer1) [reply i18n/tr.toml]: @ressonix I tried Many Thanks!

**Review by domingoruiz — APPROVED** [review]
Ok. Very good.

**Pull request merged**